server/player/player.go

```diff
@@ -1367,6 +1367,23 @@ func (p *Player) UseItem() {
 		p.updateState()
 	}
 
+	if chargeable, ok := it.(item.Chargeable); ok {
+		if !p.usingItem {
+			if !chargeable.Release(p, p.tx, p.useContext()) {
+				// Start charging the crossbow.
+				p.usingSince, p.usingItem = time.Now(), true
```

Comment:
This naming here doesn't make it very obvious as to what's happening here. `Release()` then a doc that says `"Start charging the crossbow"` is a bit confusing. Maybe try clarifying the doc a bit, to say something like "If the item was not charged yet, start charging the crossbow".